frontend/src/css/modules/contact-us.scss

```diff
@@ -0,0 +1,5 @@
+/* contact-us styles */
+
+.ev-contact-us {
```

Comment:
@Ram81 One suggestion, tryt to make a global class for this so that it can be use with other pages too like for about us page. Try to make a container with unique alias having this style. Getting?